test/e2e/traceflow_test.go

```diff
@@ -54,17 +53,77 @@ func TestTraceflow(t *testing.T) {
 	node1 := workerNodeName(1)
 	node2 := workerNodeName(2)
```

Comment:
Remember to add `skipIfNumNodesLessThan(t, 2)` at the beginning of the test. Should have been included from the get go.

Reply:
Updated.